client/client_examples/examples/python_client_usage.ipynb

```diff
@@ -44,7 +44,7 @@
     "DEFAULT_JOB_TIMEOUT = 90\n",
     "\n",
     "# sample input file and output directory\n",
-    "SAMPLE_PDF = \"/workspace/nv-ingest/data/multimodal_test.pdf\""
+    "SAMPLE_PDF = \"/workspace/data/multimodal_test.pdf\""
```

Comment:
Is this the correct path to the file? I know that it is available in data folder in the nv-ingest repo?

Reply:
I figured if the user is running the notebook outside a container, they would have to modify this path anyway, and if they are running it inside a container the path will be `/workspace/data` because we mount `./data` to `/workspace/data` inside the container:
https://github.com/NVIDIA/nv-ingest/blob/66e52b104011d8f5ea5e5e0652d6516c73a7a3eb/docker-compose.yaml#L237